pytorch_lightning/plugins/training_type/deepspeed.py

```diff
@@ -188,6 +257,17 @@ def __init__(
                 "To use the DeepSpeed plugin, you must have DeepSpeed installed."
                 " pip install deepspeed"
             )
+
+        if cpu_offload or cpu_offload_params or cpu_offload_use_pin_memory:
+            warnings.warn(
+                "The usage of ``cpu_offload`` ``cpu_offload_params`` and ``cpu_offload_use_pin_memory`` "
+                "is deprecated since v1.3 and will be removed in v1.4."
+                " From now on use ``offload_optimizer``, ``offload_parameters`` and ``pin_memory``.", DeprecationWarning
```

Comment:
Double backticks make sense for docs (as they render monospace), not for warnings

```suggestion
                "The usage of `cpu_offload`, `cpu_offload_params`, and `cpu_offload_use_pin_memory` "
                "is deprecated since v1.3 and will be removed in v1.4."
                " From now on use `offload_optimizer`, `offload_parameters` and `pin_memory`.", DeprecationWarning
```